client/src/components/SiteSearch/SiteSearchListing/SiteSearchListing.css

```diff
@@ -0,0 +1,7 @@
+.td-width-100 {
+    width: 100%;
+}
```

Comment:
@tacc-wbomar ? I think for global css we're supposed to use a naming convention that infers roles rather than dom. Something like site-search__full-width-result or something. Also, can this be in the module?

Reply:
My understanding is that these can't be in the module because these classes are being passed as props to the infinite scroll table and CSS module compilation would break that. I'll tweak the naming though.